### PR TITLE
Register 011.is-a.dev

### PR DESCRIPTION
**Pull request by iamxani**
Register 011.is-a.dev with CNAME record pointing to iamxani.github.io.

### CODE REVIEW

File: domains/011
Comment (Stef-00012) [inline domains/011:1]: file is not a JSON file (.json)

File: domains/satas
Comment (Stef-00012) [inline domains/satas:1]: file is not a JSON file (.json)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by wdhdev — DISMISSED** [review]

**Review by wdhdev — DISMISSED** [review]

**Review by wdhdev — DISMISSED** [review]

File: domains/011.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "owner": "iamxani",
```
Comment (wdhdev) [inline domains/011.json:3]:
```suggestion
    "username": "iamxani",
```

File: domains/apam.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "owner": "iamxani",
```
Comment (wdhdev) [inline domains/apam.json:3]:
```suggestion
      "username": "iamxani",
```

**Review by wdhdev — APPROVED** [review]

**Pull request merged**